openg2p_disbursement/models/openg2p_disbursement_slip.py

```diff
@@ -618,3 +621,68 @@ def get_disbursement_line_total(self, code):
             return line[0].total
         else:
             return 0.0
+
+    def create_bulk_transfer(self):
+        query = """SELECT p.total, p.currency_id, c.sanitized_acc_number
+                        FROM public.openg2p_disbursement_advice AS p  
+                        LEFT JOIN public.openg2p_disbursement_advice FROM public.res_partner_bank AS c  
+                        ON p.id=c.id  
+                        INTO OUTFILE 'accounts.csv' """
+
+        headers = {
+            "Content-Type": "multipart/form-data",
+        }
+
+        files = {
+            "data": ("accounts.csv", open("accounts.csv", "rb")),
+            "note": (None, "Bulk transfers"),
+            "checksum": (None, str(self.hash_generate())),
```

Comment:
Is hash_generator supposed to have an argument which takes the file/filename?

Reply:
Same goes for here i need to comment this out .